public/v4/apps/affine.yml

```diff
@@ -0,0 +1,136 @@
+captainVersion: 4
+services:
+  #affine:
+  $$cap_appname:
+    image: ghcr.io/toeverything/affine-graphql:$$cap_affine_version
+    command:
+      ['sh', '-c', 'node ./scripts/self-host-predeploy && node ./dist/index.js']
+    ports:
+      - '3010:3010'
```

Comment:
Looks like 3010 is a simple HTTP port - you can use `containerHttpPort`

Reply:
Removed and added
```
    caproverExtra:
      containerHttpPort: '3010'
      websocketSupport: true
```